velox/tool/trace/OperatorReplayerBase.cpp

```diff
@@ -43,10 +44,13 @@ OperatorReplayerBase::OperatorReplayerBase(
       nodeTraceDir_(exec::trace::getNodeTraceDirectory(taskTraceDir_, nodeId_)),
       fs_(filesystems::getFileSystem(taskTraceDir_, nullptr)),
       pipelineIds_(exec::trace::listPipelineIds(nodeTraceDir_, fs_)),
-      maxDrivers_(exec::trace::getNumDrivers(
-          nodeTraceDir_,
-          pipelineIds_.front(),
-          fs_)) {
+      driverIds_(
+          driverIds.empty()
+              ? exec::trace::getDriverIds(exec::trace::getNumDrivers(
```

Comment:
Can you have a new method
getDriverIds() which call getNumDrivers internally? If there is no other getNumDrivers use case, then let's change it as an internal method in trace util? thanks!